tests/test_blockchyp_client.py

```diff
@@ -0,0 +1,28 @@
+"""This file contains unit tests for the BlockChyp Client"""
+
+
+import blockchyp
+
+def test_heartbeat():
+    """This method tests functionality of the heartbeat method"""
+    creds = blockchyp.BlockChypCredentials(
+        "EN4S42E2KQL3FFQQ22K3LENPKQ",
+        "B6KV3NXFMN6YEGW2YNHS36TEVE",
+        "3fb111659a5cd66643e815db90c06bb361b50d91f97c41fe49d7cf403f343a36")
```

Comment:
These are hardcoded and will only work for your specific user. First of all, we shouldn't put credentials in source control, even if they are test credentials. Second, the test will fail if these credentials ever become invalid.

The credentials should come from a blockchyp.json or environment variables.